cmd/pd-server/main.go

```diff
@@ -44,6 +45,12 @@ func main() {
 		os.Exit(0)
 	}
 
+	defer func() {
```

Comment:
can it capture the panic if the panic happens in another goroutine? 

I suggest adding defer in all goroutines.